### DISCUSSION TITLE
[Feature Request]: 允许自定义刮削元数据&海报时使用的语言地区

### DISCUSSION THREAD

**Discussion by jimlee2048** [jxxghp/MoviePilot#3357] [讨论]
### 当前程序版本

v1.2.8

### 功能改进类型

主程序

### 功能改进

希望能够提供可配置项（环境变量），允许自定义刮削时使用的语言地区：
- 能够分别指定元数据和海报使用的语言地区。
- 可以考虑提供`auto`选项：根据影片的原始语言，刮削相应原始语言的元数据/海报
![image](https://github.com/jxxghp/MoviePilot/assets/21117946/3fee2fa9-8d49-4efd-9a75-53cc80c14e74)

### 参考资料

使用场景1：港台等地区的繁中用户或海外使用其他语言的用户，希望使用非简体中文的影片元数据。

使用场景2：元数据使用简体中文（影片简介等），但图片使用原始语言地区的海报图。
- 理由1：tmdb上简中海报质量参差不齐，常见港台版译名/杂乱的院线信息，从美观角度不如直接使用原始语言地区的海报更简单干净。例：《肖申克的救赎》
  ![image](https://github.com/jxxghp/MoviePilot/assets/21117946/2afe538b-c162-4d5e-842a-487de6d8113c)
  ![image](https://github.com/jxxghp/MoviePilot/assets/21117946/10678fa2-73cd-4b9b-a350-bce8cdd30fd9)
- 理由2：个人“原著向”的偏好，希望图片尽可能使用影片发行时使用的原始语言海报，认为看起来更“原汁原味”，且原始语言地区的电影海报往往质量更好。
- 理由3：是的，对于这个使用场景，用户完全可以自行在媒体服务器内换其他海报，但如果能通过提高自动刮削的自定义度满足更多样需求、减少用户的后续手动操作，又何乐而不为呢？:)

相关的先前issue：https://github.com/jxxghp/MoviePilot/issues/335

**Comment by 0xAmad3us** [comment]
加一，希望可以设置刮削海报的语言优先级

**Comment by sexyfeifan** [comment]
+1

**Comment by Jameszhu8** [comment]
这个需求确实可以，我现在最烦的是刮来的图片，高赞的简体海报偏偏不刮，总是下载恶心的港台译名海报，要么就直接下载原版海报好了，省得看了心烦。不过估计这跟TMDB这个网站屁股不在我们这有关，对简体用户极度不友好，建了多少词条做了多少修改全被删了要么改回去了，真特么的。

**Comment by jcc563768606** [comment]
+1

**Comment by wang-potato** [comment]
+1

**Comment by zx524724342** [comment]
刚用不就也发现了这个问题，如果可以刮削汉语区的介绍数据，封面海报这些可以手动选择为英文原版的就好了，不然台版翻译太奇怪了，例如：惩罚者 翻译为 神鬼制裁，招魂 翻译为 属阴屋

**Comment by zhuweitung** [comment]
已经被港台翻译封面整吐了，写了个脚本把所有封面都替换为了豆瓣的封面，结果尺寸比例不一致，emby上看起来不好看
摆了 :(

**Comment by gitcook** [comment]
现在是什么情况

**Comment by ing0928** [comment]
+1，台版翻译太狗血了

**Comment by maidou-00** [comment]
支持。我看到一堆“钢铁/蜘蛛 人”、“刺激1995”（肖申克救赎）、“神力女超人”（神奇女侠），真的很不适应。我曾尝试把刮削源改成豆瓣，但是仍然不行。

**Reply by maidou-00** [reply]
查了下，说是豆瓣可能会限流/控IP，因此刮削源改成豆瓣不太靠谱。

**Comment by libo26** [comment]
+1，急切需要

**Comment by cddjr** [comment]
tmdb对于封面图只有一个粗糙的汉语分类，而这分类涵盖了香港、台湾、大陆和新加坡，导致封面图只能取最近更新的，而无法获取简体中文版。

**Reply by maidou-00** [reply]
明白了，非常感谢！

**Reply by cng10001** [reply]
我看到这个类似需求19年就有了，https://www.themoviedb.org/movie/792-platoon/discuss/5d42a4b5d8cc4a001734aa84?page=1 然后至今没有实现🤦‍♂️
